### ISSUE TITLE
First access to network tab in a day is slow

### ISSUE THREAD

**Issue by lapo-luchini** [friendica/friendica#13247]
- [x] I have searched open and closed issues for duplicates

### Bug Description

Sorry if this issue is a bit vague, but it is unfortunately based on partial data: each time I visit the "/network" page for the first time in the day it is very very slow (often even timeout). On second load, it gets very fast.
To me this looks like «doing underground jobs to update it all» which makes sense… except I have this cron (which is not generating any error):
`*/10 * * * * cd /usr/local/www/friendica ; /usr/local/bin/php bin/worker.php`

### Steps to Reproduce

1. avoid web interaction for like 24h
2. visit the website interactively for the first time in a while

Actual Result:

Many seconds to generate page, or even timeout.

Expected Result:

Page load as fast as usual.

### Platform Info

Friendica Version: Friendica 'Giant Rhubarb' 2023.05 - 1518
Friendica Source: github branch 'stable'
PHP version: 7.4.32
SQL version: MariaDB

**Comment by annando** [comment]
This is not a bug. This is caused by the huge amount of unread posts that are set to "read" after the page is displayed.

**Comment by lapo-luchini** [comment]
Oh. Okay!
I'm not sure it's correct to mark them "read" to the point of not allowing me to read them by the way of a network timeout, but I understand the issue. 😇
I wonder if it would be possible to mark them read one by one, via AJAX, while I scroll the long page, instead of marking them all read on load (opening a page reading just a few and closing would leave many more as read, but not really). 🤔

**Comment by annando** [comment]
Technically a lot of stuff would be possible here. This is one of the features that could need some overhauling.

**Comment by tek-aevl** [comment]
This is normal? I have this way more frequent then it used to be..

**Comment by zedascouves** [comment]
I'm also having this problem.

Right now, after activating the bluesky connector, and deactivating it, I have 500+ unread items. How can I mark them as read, as the page doesn't even load? Directly on the database? Where exactly? :-)

**Comment by MrPetovan** [comment]
Even if you marked all the items read, I don't think it would improve the page performance, it's a parallel feature to the retrieval of items.

**Comment by zedascouves** [comment]
I see. Can I limit the number of visible items then, increasing the number of pages, for example?

**Comment by MrPetovan** [comment]
Yes, you can control the limit of displayed items in Settings -> Display -> Content/Layout. For reference, I use 15 items on desktop and 10 on mobile.

**Comment by annando** [comment]
This issue had been resolved at some point in time in the past.
